Doc/library/calendar.rst

```diff
@@ -654,6 +654,9 @@ The following options are accepted:
    The number of months printed per row.
    Defaults to 3.
 
+.. versionchanged:: 3.14
```

Comment:
I generally suggest using `next` instead of 3.14 these days. I'll yield the decision to you, though.
```suggestion
.. versionchanged:: next
```